lib/Doctrine/ORM/Tools/Export/Driver/PhpExporter.php

```diff
@@ -177,4 +179,24 @@ protected function _varExport($var)
 
         return $export;
     }
+
+    /**
+     * @param ClassMetadataInfo $metadata
+     * @param array             &$lines
+     */
+    private function processEntityListeners(ClassMetadataInfo $metadata, &$lines)
+    {
+        if (0 !== \count($metadata->entityListeners)) {
+            foreach ($metadata->entityListeners as $event => $entityListenerConfig) {
+                foreach ($entityListenerConfig as $entityListener) {
+                    $lines[] = sprintf(
+                        '$metadata->addEntityListener(\'%s\', \'%s\', \'%s\');',
+                        $event,
+                        $entityListener['class'],
```

Comment:
Where is this code tested? Can't seem to find it. Also, use `var_export()` instead of single quotes around a `%s`